### PR TITLE
Update tv.ir

### PR DESCRIPTION
**Pull request by JuicyPigWalker**
Added "TCL 50P715X1" recorded tv remote signals. Hope everything is okay, this is my first GitHub contribution.

# What's new

Added "TCL 50P715X1" recorded remote signals.

# Verification 

Data added at the bottom of the file, as per contribution guidelines.

### CODE REVIEW

File: applications/main/infrared/resources/infrared/assets/tv.ir [hunk 1]
```diff
@@ -1724,3 +1724,136 @@ type: parsed
...
+address: 0F 00 00 00
+command: 98 00 00 00
+# 
+name: Play
+type: parsed
+protocol: RCA
+address: 0F 00 00 00
+command: A8 00 00 00
+# 
+name: Stop
```
Comment (JuicyPigWalker) [inline applications/main/infrared/resources/infrared/assets/tv.ir]: Signal not supported in universal remote mode, needs to be cleared of the code.

File: applications/main/infrared/resources/infrared/assets/tv.ir [hunk 2]
```diff
@@ -1724,3 +1724,136 @@ type: parsed
...
+address: 0F 00 00 00
+command: B8 00 00 00
+# 
+name: Pause
+type: parsed
+protocol: RCA
+address: 0F 00 00 00
+command: 98 00 00 00
+# 
+name: Play
```
Comment (JuicyPigWalker) [inline applications/main/infrared/resources/infrared/assets/tv.ir]: Signal not supported in universal remote mode, needs to be cleared of the code.

File: applications/main/infrared/resources/infrared/assets/tv.ir [hunk 3]
```diff
@@ -1724,3 +1724,136 @@ type: parsed
...
+address: 0F 00 00 00
+command: 38 00 00 00
+# 
+name: Fast_ba
+type: parsed
+protocol: RCA
+address: 0F 00 00 00
+command: B8 00 00 00
+# 
+name: Pause
```
Comment (JuicyPigWalker) [inline applications/main/infrared/resources/infrared/assets/tv.ir]: Signal not supported in universal remote mode, needs to be cleared of the code.

File: applications/main/infrared/resources/infrared/assets/tv.ir [hunk 4]
```diff
@@ -1724,3 +1724,136 @@ type: parsed
...
+address: 0F 00 00 00
+command: EA 00 00 00
+# 
+name: Fast_fo
+type: parsed
+protocol: RCA
+address: 0F 00 00 00
+command: 38 00 00 00
+# 
+name: Fast_ba
```
Comment (JuicyPigWalker) [inline applications/main/infrared/resources/infrared/assets/tv.ir]: Signal not supported in universal remote mode, needs to be cleared of the code.

File: applications/main/infrared/resources/infrared/assets/tv.ir [hunk 5]
```diff
@@ -1724,3 +1724,136 @@ type: parsed
...
+address: 0F 00 00 00
+command: 6A 00 00 00
+# 
+name: Right
+type: parsed
+protocol: RCA
+address: 0F 00 00 00
+command: EA 00 00 00
+# 
+name: Fast_fo
```
Comment (JuicyPigWalker) [inline applications/main/infrared/resources/infrared/assets/tv.ir]: Signal not supported in universal remote mode, needs to be cleared of the code.

File: applications/main/infrared/resources/infrared/assets/tv.ir [hunk 6]
```diff
@@ -1724,3 +1724,136 @@ type: parsed
...
+address: 0F 00 00 00
+command: 1A 00 00 00
+# 
+name: Left
+type: parsed
+protocol: RCA
+address: 0F 00 00 00
+command: 6A 00 00 00
+# 
+name: Right
```
Comment (JuicyPigWalker) [inline applications/main/infrared/resources/infrared/assets/tv.ir]: Signal not supported in universal remote mode, needs to be cleared of the code.

File: applications/main/infrared/resources/infrared/assets/tv.ir [hunk 7]
```diff
@@ -1724,3 +1724,136 @@ type: parsed
...
+address: 0F 00 00 00
+command: 9A 00 00 00
+# 
+name: Down
+type: parsed
+protocol: RCA
+address: 0F 00 00 00
+command: 1A 00 00 00
+# 
+name: Left
```
Comment (JuicyPigWalker) [inline applications/main/infrared/resources/infrared/assets/tv.ir]: Signal not supported in universal remote mode, needs to be cleared of the code.

File: applications/main/infrared/resources/infrared/assets/tv.ir [hunk 8]
```diff
@@ -1724,3 +1724,136 @@ type: parsed
...
+address: 0F 00 00 00
+command: 2F 00 00 00
+# 
+name: Up
+type: parsed
+protocol: RCA
+address: 0F 00 00 00
+command: 9A 00 00 00
+# 
+name: Down
```
Comment (JuicyPigWalker) [inline applications/main/infrared/resources/infrared/assets/tv.ir]: Signal not supported in universal remote mode, needs to be cleared of the code.

File: applications/main/infrared/resources/infrared/assets/tv.ir [hunk 9]
```diff
@@ -1724,3 +1724,136 @@ type: parsed
...
+address: 0F 00 00 00
+command: E4 00 00 00
+# 
+name: Ok
+type: parsed
+protocol: RCA
+address: 0F 00 00 00
+command: 2F 00 00 00
+# 
+name: Up
```
Comment (JuicyPigWalker) [inline applications/main/infrared/resources/infrared/assets/tv.ir]: Signal not supported in universal remote mode, needs to be cleared of the code.

File: applications/main/infrared/resources/infrared/assets/tv.ir [hunk 10]
```diff
@@ -1724,3 +1724,136 @@ type: parsed
...
+address: 0F 00 00 00
+command: 86 00 00 00
+# 
+name: Info
+type: parsed
+protocol: RCA
+address: 0F 00 00 00
+command: 3C 00 00 00
+# 
+name: Back
```
Comment (JuicyPigWalker) [inline applications/main/infrared/resources/infrared/assets/tv.ir]: Signal not supported in universal remote mode, needs to be cleared of the code.

File: applications/main/infrared/resources/infrared/assets/tv.ir [hunk 11]
```diff
@@ -1724,3 +1724,136 @@ type: parsed
...
+address: 0F 00 00 00
+command: 37 00 00 00
+# 
+name: List
+type: parsed
+protocol: RCA
+address: 0F 00 00 00
+command: 86 00 00 00
+# 
+name: Info
```
Comment (JuicyPigWalker) [inline applications/main/infrared/resources/infrared/assets/tv.ir]: Signal not supported in universal remote mode, needs to be cleared of the code.

File: applications/main/infrared/resources/infrared/assets/tv.ir [hunk 12]
```diff
@@ -1724,3 +1724,136 @@ type: parsed
...
+address: 0F 00 00 00
+command: C5 00 00 00
+# 
+name: Menu
+type: parsed
+protocol: RCA
+address: 0F 00 00 00
+command: 37 00 00 00
+# 
+name: List
```
Comment (JuicyPigWalker) [inline applications/main/infrared/resources/infrared/assets/tv.ir]: Signal not supported in universal remote mode, needs to be cleared of the code.

File: applications/main/infrared/resources/infrared/assets/tv.ir [hunk 13]
```diff
@@ -1724,3 +1724,136 @@ type: parsed
...
+address: 0F 00 00 00
+command: F3 00 00 00
+# 
+name: Input
+type: parsed
+protocol: RCA
+address: 0F 00 00 00
+command: C5 00 00 00
+# 
+name: Menu
```
Comment (JuicyPigWalker) [inline applications/main/infrared/resources/infrared/assets/tv.ir]: Signal not supported in universal remote mode, needs to be cleared of the code.

File: applications/main/infrared/resources/infrared/assets/tv.ir [hunk 14]
```diff
@@ -1724,3 +1724,136 @@ type: parsed
...
+address: 0F 00 00 00
+command: 54 00 00 00
+# 
+name: Setup
+type: parsed
+protocol: RCA
+address: 0F 00 00 00
+command: F3 00 00 00
+# 
+name: Input
```
Comment (JuicyPigWalker) [inline applications/main/infrared/resources/infrared/assets/tv.ir]: Signal not supported in universal remote mode, needs to be cleared of the code.

File: applications/main/infrared/resources/infrared/assets/tv.ir [hunk 15]
```diff
@@ -1724,3 +1724,136 @@ type: parsed
...
 command: 21 00 00 00
+#
+#TCL 50P715X1
+name: Power
+type: parsed
+protocol: RCA
+address: 0F 00 00 00
+command: 54 00 00 00
+# 
+name: Setup
```
Comment (JuicyPigWalker) [inline applications/main/infrared/resources/infrared/assets/tv.ir]: Signal not supported in universal remote mode, needs to be cleared of the code.

**Review by skotopes — DISMISSED** [review]

**Review by skotopes — APPROVED** [review]

**Pull request merged**